cmd/cpe2cve/cpe2cve.go

```diff
@@ -34,6 +37,33 @@ import (
 	"github.com/facebookincubator/nvdtools/wfn"
 )
 
+// input a
+func processSingle(caches map[string]*cvefeed.Cache, cfg config, cpe string) []string {
+	var cves = []string{}
+	attr, err := wfn.Parse(cpe)
+	if err != nil {
+		flog.Errorf("couldn't parse uri %q: %v", cpe, err)
+		return []string{}
+	}
+	cpes := make([]*wfn.Attributes, 0, 1)
+	cpes = append(cpes, attr)
+	for _, cache := range caches {
```

Comment:
It would be useful to expose the key of `caches` in the output JSON, IIRC, in case feeds from multiple data providers are used.